File: plusmall-operator-interface/src/main/java/com/plusmall/operator/BrandService.java
package com.plusmall.operator;

import com.plusmall.commons.PageResult;
import com.plusmall.model.TbBrand;

import java.util.List;
import java.util.Map;

/**
 * @Description:
 */
public interface BrandService {
	public void add(TbBrand brand);
	public TbBrand getBrandById(Long id);
	public void update(TbBrand brand);
	public void delete(Long[] ids);
	public PageResult searchBrands(TbBrand brandinfo,int pageNum,int pageSize);
	public List<Map> selectOptionList();
}

File: plusmall-operator-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {
    //删除
    this.delete=function (ids) {
        return $http.get('../brand/delete.do?ids='+ids);
    }
    
    //查找
    this.search=function (pageNum, pageSize, searchEntity) {
        return $http.post('../brand/search.do?pageNum='+pageNum+'&pageSize='+pageSize,searchEntity);
    }

    //新增
    this.add=function (brandEntity) {
        return $http.post('../brand/addBrand.do',brandEntity);
    }

    //修改
    this.update=function (brandEntity) {
        return $http.post('../brand/updateBrand.do',brandEntity)
    }

    //根据id查找单个brand数据
    this.findOne=function (id) {
        return $http.get('../brand/findOne.do?id='+id);
    }
});